### PR TITLE
[Feature][WRS-1753] Extend "MediaConnectorCapabilities" with new "metadata" field

### PR DESCRIPTION
**Pull request by psamusev**
This PR

## PR Guidelines

- [ ] I have read the [contribution and PR guidelines](/chili-publish/studio-sdk/blob/main/CONTRIBUTING.md)

## Related tickets

-   [WRS-1753](https://support.chili-publish.com/projects/WRS/issues/WRS-1753)

## Screenshots

[WRS-1753]: https://chilipublishintranet.atlassian.net/browse/WRS-1753?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by abdelhalimkhouas — APPROVED** [review]

File: packages/connectors/src/MediaConnector.ts [hunk 1]
```diff
@@ -42,4 +42,5 @@ export type MediaConnectorCapabilities = {
     query: boolean;
     detail: boolean;
     filtering: boolean;
+    metadata?: boolean;
```
Comment (Matthiee) [inline packages/connectors/src/MediaConnector.ts:45]: Is this nullable to make it backwards compatible?

Comment (psamusev) [reply packages/connectors/src/MediaConnector.ts:45]: Yes

Comment (Matthiee) [reply packages/connectors/src/MediaConnector.ts:45]: This will not work without engine changes, as these capabilities are typed so they will get lost.

Comment (brapoprod) [reply packages/connectors/src/MediaConnector.ts:45]: @Matthiee can we make sure to fix this? connectors are public for a month or so now so we basically can't do breaking changes anymore

**Review by Matthiee — APPROVED** [review]
LGTM, keep in mind this new field will always be undefined until it is supported on engine side.

**Pull request merged**